lib/pages/login_page.dart

```diff
@@ -40,44 +42,55 @@ class LoginPage extends StatelessWidget {
     );
   }
 
-  Widget _buildLoginForm() {
+  Widget _buildLoginForm(BuildContext context) {
+    final forgotButtonWidth = 80.0;
+
     return Column(
       mainAxisAlignment: MainAxisAlignment.center,
       children: [
-        CupertinoTextField(
+        TextField(
           autofocus: true,
-          keyboardType: TextInputType.emailAddress,
-          maxLength: 100,
           // TODO localization https://github.com/luongvo/flutter-survey/issues/25
-          placeholder: "Email",
+          decoration: _formInputDecoration(context, "Email"),
+          keyboardType: TextInputType.emailAddress,
+          style: Theme.of(context).textTheme.bodyText1,
           textInputAction: TextInputAction.next,
         ),
         SizedBox(
           height: 20.0,
         ),
         Stack(
           children: [
-            CupertinoTextField(
-              maxLength: 100,
+            TextField(
+              decoration: _formInputDecoration(context, "Password").copyWith(
+                contentPadding: EdgeInsets.symmetric(
+                  horizontal: Dimens.inputHorizontalPadding,
+                  vertical: Dimens.inputVerticalPadding,
+                ).copyWith(
+                  right: forgotButtonWidth,
+                ),
+              ),
               obscureText: true,
-              placeholder: "Password",
+              obscuringCharacter: "●",
+              style: Theme.of(context).textTheme.bodyText1,
               textInputAction: TextInputAction.done,
             ),
             Positioned.fill(
               child: Align(
                 alignment: Alignment.centerRight,
-                child: TextButton(
-                  child: Text(
-                    "Forgot?",
-                    style: TextStyle(
-                      color: Colors.black,
-                      fontSize: 15,
-                      fontWeight: FontWeight.normal,
+                child: SizedBox(
+                  height: double.infinity,
```

Comment:
just curious if this is necessary? 🤔

Reply:
@sleepylee yes, to keep the button fits its parent for better touch effect
![image](https://user-images.githubusercontent.com/16315358/132207346-b42cacc2-0789-47ed-90fc-3186fcfa47a5.png)